### PR TITLE
Deploy to Heroku

### PR DESCRIPTION
**Pull request by l1storez**

### CODE REVIEW

File: djangogirls/mysite/settings.py [hunk 1]
```diff
@@ -130,4 +136,7 @@
 # Default primary key field type
 # https://docs.djangoproject.com/en/3.2/ref/settings/#default-auto-field
 
-DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
```
Comment (webknjaz) [inline djangogirls/mysite/settings.py:133]: recover this

**Comment by l1storez** [comment]
Split commit ddaef4c into several atomic?

**Comment by webknjaz** [comment]
Why is 0253acb9d766b3ff1e9a3f5dde8efd9d21ff6c9c separate?

**Comment by webknjaz** [comment]
> Split commit [ddaef4c](https://github.com/kpi-web-guild/django-girls-blog-l1storez/commit/ddaef4c48125fc27036dded40fd8d4587c1345fb) into several atomic?

I'd fix the commit message to be useful at least. But maybe. It's currently unclear what it's supposed to do from the message having a list of things that don't represent any specific changes.

**Comment by webknjaz** [comment]
Also, what's with the PR description?

File: djangogirls/mysite/settings.py [hunk 2]
```diff
@@ -26,9 +27,9 @@
 )
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
 
-ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com']
+ALLOWED_HOSTS = ['127.0.0.1', '.pythonanywhere.com', '.heroku.app']
```
Comment (webknjaz) [inline djangogirls/mysite/settings.py:32]: Why `.app`? I haven't heard of them using this domain. Can you show me a document that describes such a possibility?

**Comment by l1storez** [comment]
> Why is [0253acb](https://github.com/kpi-web-guild/django-girls-blog-l1storez/commit/0253acb9d766b3ff1e9a3f5dde8efd9d21ff6c9c) separate?

In GHA happened dependency incompatibility

File: djangogirls/mysite/settings.py (hunk 2 above)
Comment (l1storez) [reply djangogirls/mysite/settings.py:32]: It's my mistake

**Review by webknjaz — CHANGES_REQUESTED** [review]
It looks like you've failed to follow this step: http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise

**Comment by webknjaz** [comment]
> > Why is [0253acb](https://github.com/kpi-web-guild/django-girls-blog-l1storez/commit/0253acb9d766b3ff1e9a3f5dde8efd9d21ff6c9c) separate?
> 
> In GHA happened dependency incompatibility

So what?

**Comment by l1storez** [comment]
> So what?

I did not rebase it because I was waiting for an answer to the question about the first commit.  As there are options or squash with the first commit or make a new commit(for example "add dependencies for Heroku) and squash with it

**Comment by l1storez** [comment]
> It looks like you've failed to follow this step: http://whitenoise.evans.io/en/stable/django.html#enable-whitenoise

This step is not in the tutorial. But if it is necessary, I will add

**Comment by webknjaz** [comment]
> This step is not in the tutorial. But if it is necessary, I will add

The tutorial does not use the version of Whitenoise that you integrated.
Also, if you want to check if your integration actually works, just run the command from the `web` section in `Procfile` locally.

**Comment by l1storez** [comment]
> Also, if you want to check if your integration actually works, just run the command from the `web` section in `Procfile` locally.

Okay, thanks

**Comment by l1storez** [comment]
> This step is not in the tutorial. But if it is necessary, I will add

Done!

**Comment by webknjaz** [comment]
Please make the commit messages useful and atomic. Changing and adding files without a specified purpose is weird.

**Comment by webknjaz** [comment]
> This PR describes the steps

It doesn't describe anything at all

**Comment by l1storez** [comment]
> It doesn't describe anything at all

Okay:)

**Review by webknjaz — APPROVED** [review]

**Comment by l1storez** [comment]
Oh, wow

**Pull request merged**